src/Evolution/Systems/GeneralizedHarmonic/Tags.hpp

```diff
@@ -222,11 +291,11 @@ namespace OptionTags {
  * *rolled-on* to) at the beginning of an evolution. This parameter sets
  * the coordinate time at which roll-on begins.
  */
-struct GaugeHRollOnStartTime : db::SimpleTag {
+struct GaugeRollOnStart {
```

Comment:
Do you mean something like:

```
namespace OptionTags {                                                          
/*!                                                                             
 * \ingroup OptionGroupsGroup                                                   
 * Groups option tags related to the GeneralizedHarmonic evolution system.      
 */                                                                             
struct GeneralizedHarmonicGroup {                                               
  static std::string name() noexcept { return "GeneralizedHarmonic"; }          
  static constexpr OptionString help{"Options for the evolution system"};       
  using group = ::OptionTags::EvolutionSystemGroup;                             
};                                                                              
                                                                      
struct GaugeGroup {                                               
  static std::string name() noexcept { return "GeneralizedHarmonic"; }          
  static constexpr OptionString help{"Options for the evolution system"};       
  using group = GeneralizedHarmonic::OptionTags::GeneralizedHarmonicGroup;                             
}; 

/*!                                                                             
 * \ingroup OptionTagsGroup                                                     
 * \brief Gauge control parameter determining when to start rolling-on the      
 * evolution gauge.                                                             
 *                                                                              
 * \details The evolution gauge is gradually transitioned to (or                
 * *rolled-on* to) at the beginning of an evolution. This parameter sets        
 * the coordinate time at which roll-on begins.                                 
 */                                                                             
struct RollOnStart {                                                      
  using type = double;                                                          
  static constexpr OptionString help{                                           
      "Simulation time to start rolling-on evolution gauge"};                   
  using group = GaugeGroup;                                       
};                                                                              
                                                                                
struct RollOnWindow {                                                     
  using type = double;                                                          
  static constexpr OptionString help{                                           
      "Duration of gauge roll-on in simulation time"};                          
  using group = GaugeGroup;                                       
};                                                                              

template <typename Frame>                                                       
struct SpatialDecayWidth {                                                
  using type = double;                                                          
  static constexpr OptionString help{                                           
      "Spatial width of weighting factor in evolution gauge"};                  
  using group = GaugeGroup;                                       
};          
```

Reply:
Yep, just keep the `Gauge` in the struct names: `struct GaugeRollOnStart { ... }`

Reply:
Okay, cool. Just to make sure I'm reading this right, now there's Gauge in struct names but not in their `name()`. Which means the `name` method gets added back everywhere. Yes?

Reply:
Also, do you want "Gauge" in their names or "GaugeH"?

Reply:
- Yes, gauge in struct names (since they are not in an extra namespace and not length-constrained), and yes you need a name method for the option tags in the gauge group
- Just make Gauge vs GaugeH consistent between tag names :)

Reply:
done.